integration-tests/ldap/src/main/java/org/apache/camel/quarkus/component/ldap/it/LdapResource.java

```diff
@@ -62,43 +70,43 @@ public class LdapResource {
     @POST
     @Consumes(MediaType.APPLICATION_JSON)
     public void configure(Map<String, String> options) throws Exception {
-        String host = options.get("host");
-        String port = options.get("port");
-        boolean useSSL = Boolean.valueOf(options.get("ssl"));
-        String trustStoreFilename = options.get("trustStore");
-        String trustStorePassword = options.get("trustStorePassword");
-
-        DirContext dirContext = createLdapContext(host, port, useSSL, trustStoreFilename, trustStorePassword);
-        camelContext.getRegistry().bind("ldapserver", dirContext);
+        ldapHost = options.get("host");
+        ldapPort = options.get("port");
+        useSSL = Boolean.valueOf(options.get("ssl"));
+        trustStoreFilename = options.get("trustStore");
+        trustStorePassword = options.get("trustStorePassword");
     }
 
     @Path("/search")
     @GET
     @Produces(MediaType.APPLICATION_JSON)
-    public Response search(@QueryParam("q") String filter) throws Exception {
-        ProducerTemplate producer = camelContext.createProducerTemplate();
-        List<SearchResult> results = producer.requestBody("direct:start", filter, List.class);
-        return Response.ok(convertSearchResults(results)).build();
+    public Response search(@QueryParam("q") String q) throws Exception {
```

Comment:
This `q` param seems mysterious. Could we find a better name ? Or maybe `q` is a kind of LDAP notion ?

Reply:
I guess it's short for 'query'.

Given Darren is in PTO, I'd vote for merging this and following up later if we really need to.

Reply:
Ok, might be ldapQuery. Ok, let's do this.